subprojects/distributions/src/changes/accepted-public-api-changes.json

```diff
@@ -165,6 +165,51 @@
             "type": "org.gradle.api.tasks.testing.AbstractTestTask",
             "member": "Constructor org.gradle.api.tasks.testing.AbstractTestTask()",
             "acceptation": "Extracted super class out of Test which was already stable"
-        }
+        },
+        {
+            "type": "org.gradle.api.tasks.testing.ExecuteTestBuildOperationType",
+            "member": "Class org.gradle.api.tasks.testing.ExecuteTestBuildOperationType",
+            "acceptation": "Build Operation typs are not annotated with incubating"
```

Comment:
:x: typo `typs` -> `types`, same in others

Reply:
as we made those build operation types internal I removed them from the binary check json file completely